app/controllers/competitions_controller.rb

```diff
@@ -610,7 +610,7 @@ def user_preferences
     render json: competition.form_user_preferences(current_user)
   end
 
-  before_action -> { require_user_permission(:can_manage_competition?, competition_from_params) }, only: [:announcement_data]
+  before_action -> { require_user_permission(:can_manage_competition?, competition_from_params) }, only: [:confirmation_data]
```

Comment:
Unrelated change - I noticed this when I was playing around with headers. Happy to remove and put in a separate PR if necessary, but it seems like a clear copy/paste error to me - looking at the full context of the file should make this obvious (`announcement_data` already has an identical before_action elsewhere, but `confirmation_data` doens't)